docs/FunctionalTestBuildScripts.md

```diff
@@ -204,6 +204,14 @@ The Visual Studio 2022 "empty console application" template, compiled as Release
 
 The Visual Studio 2022 "empty console application" template, compiled as Release|x64.  The `/incremental:no` option disables incremental linking explicitly.  The `/ltcg` and `/gl` options are disabled explicitly.
 
+## Native_x64_VS2022_IntegrityCheck_[variant].exe
```

Comment:
yes, the flag works in driver!
I will keep the exe test files because exe is easy to verify just double click Windows will do verification, and will block it due to verification failed.